lib/mayaUsd/python/wrapMeshWriteUtils.cpp

```diff
@@ -70,13 +70,16 @@ class DummyScopeClass{};
 
 } // anonymous namespace 
 
-
 void wrapMeshWriteUtils()
 {
+    // TODO: HS, June 11, 2020 This logics is not accurate. Revisit this later.
+    // There are no test cases around is not being consumed in any tests.
+    /*
     scope s = class_<DummyScopeClass>("MeshWriteUtils", no_init)
 
         .def("GetMeshNormals", &_GetMeshNormals)
             .staticmethod("GetMeshNormals")
 
         ;
+    */
```

Comment:
@HamedSabri-adsk: Can you please revert this change? True that the Python bindings for this are not used anywhere in maya-usd, but we have internal code that does this (using the old name):

```Python
from pxr.UsdMaya import MeshUtil
(normalsArray, interpolation) = MeshUtil.GetMeshNormals(...)
```

Reply:
Hey @mattyjams  Sure will do. But before that I want to understand a few things:

1-
I can see in the original code that GetMeshNormals is not a static function and yet it in the python binding is declared as `staticmethod`.  I am not an expert in Boost python binding but curious how this works?

https://github.com/PixarAnimationStudios/USD/commit/be80accecb33109fcc40471b4089c3748f50abc1

2- MeshUtil has now renamed MeshWriteUtils. I had to fix it in this commit:

https://github.com/Autodesk/maya-usd/pull/420/commits/a520d00fd70cac0421f5799e23bdbee5230935eb

Does this mean that we should revert these name changes as well?

Reply:
For #1, we're binding `MeshWriteUtils.GetMeshNormals()` in Python to the `_GetMeshNormals()` function declared just above in `wrapMeshWriteUtils.cpp`, which is indeed declared static, and returns a boost tuple type:
https://github.com/Autodesk/maya-usd/blob/5803a4695f23006e8f8367c8422e32c14601d6d1/lib/mayaUsd/python/wrapMeshWriteUtils.cpp#L42

For #2, we do want the name changes so that files/classes/etc. are all in alignment, so that part should **not** be reverted. We can absorb the name change no problem, we just need a function with the new name to exist so that we can switch to it. :P

Reply:
@mattyjams Gotcha...  see be3c5450b31d787eb4616a084b739b8eabfa41f8

Reply:
Great, thanks!